docs/scheduler-config.md

```diff
@@ -563,7 +567,7 @@ namespace provided, or the default if it is not set.</p>
 <code>map</code></br>
 <em>
 <a href="#scheduler.jenkins-x.io/v1alpha1.ConfigMapSpec">
-map[string]github.com/jenkins-x-plugins/jx-gitops/pkg/apis/scheduler/v1alpha1.ConfigMapSpec
+map[string]./pkg/apis/scheduler/v1alpha1.ConfigMapSpec
```

Comment:
@ankitm123 mmm. i got this the first run
```
make docs
....
github.com/jenkins-x-plugins/jx-gitops/pkg/cmd
command-line-arguments
GO111MODULE=on go get github.com/jenkins-x/gen-crd-api-reference-docs
go: downloading github.com/jenkins-x/gen-crd-api-reference-docs v0.1.6
go: downloading k8s.io/klog v1.0.0
go: downloading k8s.io/gengo v0.0.0-20211129171323-c02415ce4185
go: downloading github.com/russross/blackfriday/v2 v2.1.0
gen-crd-api-reference-docs -config "hack/configdocs/config.json" \
	-template-dir hack/configdocs/templates \
    -api-dir "./pkg/apis/gitops/v1alpha1" \
    -out-file docs/config.md
/bin/sh: gen-crd-api-reference-docs: command not found
make: *** [generate-refdocs] Error 127
```

so i followed the https://github.com/tektoncd/ahmetb-gen-crd-api-reference-docs readme
```
git clone https://github.com/tektoncd/ahmetb-gen-crd-api-reference-docs
cd ahmetb-gen-crd-api-reference-docs
go build
mv gen-crd-api-reference-docs /usr/local/bin
chmod +x /usr/local/bin/gen-crd-api-reference-docs
```
am i breaking anything here?

Reply:
Well, probably not you. I don't know who did. For now just leave out the doc changes that doesn't stem from your other changes.

Reply:
done